src/bundle/Resources/public/js/scripts/admin.content.tree.js

```diff
@@ -1,98 +1,26 @@
-(function (global, doc, React, ReactDOM, eZ, localStorage) {
-    const KEY_CONTENT_TREE_EXPANDED = 'ez-content-tree-expanded';
-    const CLASS_CONTENT_TREE_EXPANDED = 'ez-content-tree-container--expanded';
-    const CLASS_CONTENT_TREE_ANIMATE = 'ez-content-tree-container--animate';
-    const CLASS_BTN_CONTENT_TREE_EXPANDED = 'ibexa-btn--content-tree-expanded';
+(function (global, doc, React, ReactDOM, eZ) {
     const token = doc.querySelector('meta[name="CSRF-Token"]').content;
     const siteaccess = doc.querySelector('meta[name="SiteAccess"]').content;
-    const contentTreeContainer = doc.querySelector('.ez-content-tree-container');
-    const contentTreeWrapper = doc.querySelector('.ez-content-tree-container__wrapper');
-    const btn = doc.querySelector('.ibexa-btn--toggle-content-tree');
+    const contentTreeContainer = doc.querySelector('.ibexa-content-tree-container');
```

Comment:
Question: 
Maybe we can change the name of the class because for me `wrapper` and `container` are interchangeable terms for HTML node that groups other nodes or some content